playwright.config.ts

```diff
@@ -38,15 +38,15 @@ export default defineConfig({
       use: { ...devices['Desktop Chrome'] },
     },
 
-    {
+    /*{
```

Comment:
Pourquoi avoir commenté ça ?

Reply:
pour itérer plus rapidement, je les remets